setup.py

```diff
@@ -40,6 +40,8 @@
     "PyYAML>=5.1,<6.0",
     "semver>=2.10.2",
     "packaging>=20.4",
+    "pandas>=1.1.5",
```

Comment:
Please put them in a new target `bench`, so that people who wouldn't like to install benchmark don't have to get pandas etc. And plug it under `all`, so that we could do the usual `pip install '.[all]'`  to have both r-dev and r-bench.

Reply:
What will happen if the user didn't install `benchmark` target but entry point `reana-benchmark` is still created?

Reply:
Actually the benchmarking script could run partially, i.e. launching workflows and collecting results, only the visualisation part would fail when matplotlib is not there... Kind of like yadage uses `viz`...

Reply:
Got it. If this is not a big issue and we can move this PR forward, I am fine with that.

BTW, added `pandas` and `matplotlib` to `benchmark` target.